### ISSUE TITLE
Quiver empties with over one stack of arrows

### ISSUE THREAD

**Issue by macf2802** [Strubbelkopp/Quiver#5]
I made a video to demonstrate the bug: https://imgur.com/a/SOUeFcY
The same happens in survival mode.

**Comment by Strubbelkopp** [comment]
Hmm, do you have any other mods installed? The problem I had is that the item amount in a stack is normally saved in a `byte`, which allows for a maximum size of 127. To solve that I made it, so the stack size get's saved in a `short` instead, which isn't elegant, but it works.
The only way I could explain this happening, is some conflict with another mod. Could you send your log file?

**Comment by macf2802** [comment]
Yes I do have other mods installed. I'll try to pinpoint which one is causing the issue.
